Makefile

```diff
@@ -8,3 +8,9 @@ lint:
 
 dev:
 	$(SRC)/manage.py runserver 127.0.0.1:8000
+
+builddeb:
+	dpkg-buildpackage -us -uc -b
```

Comment:
debuild is a good idea, thanks

auto-changelogging would be cool, will investigate that when we have Jenkins or something setup since it would let us run it automatically on push